unittest/srdf.cpp

```diff
@@ -88,9 +88,9 @@ BOOST_AUTO_TEST_CASE(readReferenceConfig_stream)
   pinocchio::srdf::loadReferenceConfigurationsFromXML(model,iss,false);
 
   Eigen::VectorXd q = model.referenceConfigurations["reference"];
-  Eigen::VectorXd qexpected (2); qexpected << 1,0;
+  Eigen::VectorXd qexpected(1); qexpected << 1;
```

Comment:
The bug is somewhere else then. The size of q should be 2...

Reply:
You put a revolute model, not a continuous one.

Reply:
You are right. I though a revolute joint without bounds was a continuous joint but it is a mistake.

I meant "continuous" as it will also check the loadReferenceConfigurations handles continuous joints properly.

BTW, I am surprised this test succeeded before. qexpected should have been 0, not 1...

Reply:
> BTW, I am surprised this test succeeded before. qexpected should have been 0, not 1...

understood:
```
BOOST_CHECK(!q.isApprox(qexpected));
```

Reply:
Yes, I changed the value by forced-pushed.

Reply:
I think to test the continuous case, we should add an additional test.

Reply:
I've just done it.